techniques/failures/F107.html

```diff
@@ -2,125 +2,94 @@
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
-         </p>
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://w3c.github.io/wcag/understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。</p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>This technique applies to <em>form inputs</em> created in HTML which collect information about the user of the form.
-            </p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/identify-input-purpose">Success Criterion 1.3.5: Identify Input Purpose</a> (Failure)</span>.
-            </p>
+            <h2>適用 (対象)</h2>
+            <p>この達成方法は、フォーム利用者の情報を集める HTML で作られた<em>入力フィールド</em>に適応される。</p>
+            <p>これは、<span><a href="https://waic.jp/docs/WCAG21/Understanding/identify-input-purpose">達成基準 1.3.5: 入力目的の特定</a>（十分）</span>に関連する達成方法である。</p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The purpose of this technique is to identify a failure condition where form inputs
-               do not have the correct autocomplete attribute values for inputs that request information
-               about the user of the form.
-            </p>
-            <p>Success Criteria 1.3.5 uses a fixed list of tokens in <a href="https://www.w3.org/TR/WCAG21/#input-purposes">Input Purposes for User Interface Components</a> (based on the HTML 5.2 <code>autocomplete</code> attribute’s fixed list of token values) because the programmatic association of specified
-               token values (metadata) allows for other machine processing, such as expressing the
-               input label in different modalities.
-            </p>
-            <p>Another important part of this Success Criterion is that the token values are associated
-               with inputs that are <strong>scoped directly to the primary end user</strong>.
-            </p>
+            <h2>解説</h2>
+            <p></p>
+            <p>この達成方法の目的は、フォームの利用者に関する情報を要求する入力フィールドにおいて、入力フィールドが正しい autocomplete 属性値を持っていないという失敗条件を特定することにある。</p>
+            <p>達成基準 1.3.5 で <a href="https://www.w3.org/TR/WCAG21/#input-purposes">Input Purposes for User Interface Components</a> の（HTML 5.2 の <code>autocomplete</code> 属性のトークン値の固定リストに基づく）トークンの固定リストを用いるのは、トークン値（メタデータ）をプログラム的に関連づけることで、入力ラベルを異なるモダリティで表現する等の機械的な処理を可能にするためである。</p>
```

Comment:
`modality` という単語は「法性」「様相性」 ([参考](https://ejje.weblio.jp/content/modality)) などと直訳されますが，このコンテキストにおいては不適切 / カタカナの「モダリティ」がやや浸透しつつあることからそれに落ち着きました．

原文：
> ... such as expressing the input label in different modalities

翻訳：
> 入力ラベルを異なるモダリティで表現する等の...

Reply:
```suggestion
            <p>達成基準 1.3.5 で (HTML 5.2 の <code>autocomplete</code> 属性のトークン値の固定リストに基づく) <a href="https://waic.jp/docs/WCAG21/#input-purposes">ユーザインタフェース コンポーネントの入力目的</a> のトークンの固定リストを使用する。これは、トークン値（メタデータ）をプログラム的に関連づけることで、入力ラベルを異なるモダリティで表現する等の機械的な処理を可能にするためである。</p>
```
WAICのリソースに置き換えつつ、2文に分割しました。